package.json

```diff
@@ -20,7 +20,7 @@
     "babel-core": "^6.24.1",
     "babel-loader": "^6.2.x",
     "babel-preset-es2015": "^6.24.1",
-    "backbone": "~1.1.2",
+    "backbone": "~1.3.3",
```

Comment:
I don't think moving it to a separate PR is necessary. I just wanted to make sure we were aware of any new features we could use and/or cleanup we could do, and it seems like you've done that.